packages/config/src/env/envelope.js

```diff
@@ -0,0 +1,25 @@
+import { throwUserError } from '../error.js'
+import { ERROR_CALL_TO_ACTION } from '../log/messages.js'
+
+export const getEnvelope = async function ({ api, accountId, siteId }) {
+  if (accountId === undefined) {
+    return {}
+  }
+  try {
+    const environmentVariables = await api.getEnvVars({ accountId, siteId })
+    // eslint-disable-next-line fp/no-mutating-methods
+    const envVars = environmentVariables
```

Comment:
[dust] I wonder if we could name this variable in a way that describes how `envVars` differs from `environmentVariables`. If I understand this correctly, we're sorting the variables and filtering them by context. Can we come up with a name that somehow captures that? 🤔